packages/apps-config/src/api/spec/realis.ts

```diff
@@ -30,9 +30,10 @@ const definitions: OverrideBundleDefinition = {
             'Diamond'
           ]
         },
+        String: 'Vec<u8>',

```

Comment:
Would suggest “Text” - it extends the JS string type.

Reply:
In blockchain, we use String

Reply:
I understand. On the JS API the “Text” class maps to the Rust string type. 

(String is already a class in JS, hence the rename)